README.md

```diff
@@ -18,6 +18,82 @@ to facilitate unit-testing of OpenTracing Java instrumentation.
 
 Packages are deployed to Maven Central under the `io.opentracing` group.
 
+## Usage
```

Comment:
There was basically nothing in this README for users of OT-Java. Now there's *something*, though it's too heavily weighted towards span scheduling :-/